src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1083,8 +1083,20 @@ public void dump(final byte[] key) {
     sendCommand(DUMP, key);
   }
 
-  public void restore(final byte[] key, final int ttl, final byte[] serializedValue) {
-    sendCommand(RESTORE, key, toByteArray(ttl), serializedValue);
+  public void restore(final byte[] key, final int ttl, final byte[] serializedValue, final String... replaceArgs) {
```

Comment:
Hmmm I don't like adding varargs just to make the replace parameter optional, it makes the interface less clear. I believe we should add a new `restore` method that makes this explicit.